feeds/components.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "name": "Preview.ConversationalCore.GeneralIntent",
+    "version": "0.0.1-preview1",
```

Comment:
version feels like it ought to be an array or capable of representing version ranges. Or indeed, do we even need this, and instead just pull all available versions available from the backing package source?

Reply:
This is should be understood as the latest version. Users can install older versions by hand.

Eventually this data will come from nuget or npm, where there will be a single version represented in the feed.

Reply:
This _can be_ optional in the feed fwiw, in which case it pulls the latest version.

Reply:
however listing the version makes the table look better in Composer ;)

Reply:
I think I'm even more confused then - given my other comment, as I've understood the response data for retrieving package metadata from both npm and NuGet, I believe they largely contain all of the available versions. At that point, would we not just display the latest version as available directly from the source?